cron-jobs/checkForLastLogin.js

```diff
@@ -4,26 +4,32 @@ const emailService = require('~/services/email')
 const emailSubject = require('~/consts/emailSubject')
 const { oneDayInMs } = require('~/consts/auth')
 
```

Comment:
Add unit tests.

Reply:
I have added tests for remove unverified users cron job.